stripe-crypto-onramp/src/main/java/com/stripe/android/crypto/onramp/di/OnrampModule.kt

```diff
@@ -45,6 +48,16 @@ internal class OnrampModule {
         return PaymentConfiguration.getInstance(appContext)
     }
 
+    @Provides
+    @Named(PUBLISHABLE_KEY)
+    fun providePublishableKey(paymentConfiguration: Provider<PaymentConfiguration>): () -> String =
+        { paymentConfiguration.get().publishableKey }
```

Comment:
👀 We do it this way so if `PaymentConfiguration.init()` is called again, the new value is used in subsequent Crypto API calls.